irohad/ametsuchi/impl/storage_impl.cpp

```diff
@@ -406,13 +406,15 @@ namespace iroha {
       try {
         *(storage->sql_) << "COMMIT";
         storage->committed = true;
-        return createPeerQuery() |
-            [](const auto &peer_query) { return peer_query->getLedgerPeers(); }
-        | [](auto &&peers) {
-            return boost::optional<std::unique_ptr<LedgerState>>(
-                std::make_unique<LedgerState>(
-                    std::make_shared<PeerList>(std::move(peers))));
-          };
+        return PeerQueryWsv(std::make_shared<PostgresWsvQuery>(*(storage->sql_),
+                                                               factory_))
+                   .getLedgerPeers()
```

Comment:
Also, as we investigate there is an issue with `PeerQueryWsv`. `GetLedgerPeers` method could be simplified. I suggest to rework it in this pr or create a separate one.